srcpkgs/retroarch/template

```diff
@@ -73,9 +72,9 @@ if [ "$build_option_wayland" ]; then
 	if [ -z "$build_option_gles2" -a -z "$build_option_glcore" ]; then
 		msg_error "$pkgname: 'wayland' option requires 'gles2' or 'glcore'.\n"
 	fi
-	makedepends+=" wayland-devel wayland-protocols"
+	makedepends+=" wayland-devel wayland-protocols libdecor-devel"
 	if [ "$CROSS_BUILD" ]; then
-		hostmakedepends+=" wayland-devel wayland-protocols"
+		hostmakedepends+=" wayland-devel wayland-protocols libdecor-devel"
```

Comment:
Cross-building probably requires `wayland-devel` and `wayland-protocols` here because the `wayland-scanner` executable is run on the host. I doubt `libdecor-devel` is required on the host as well.

Reply:
Fixed. Thanks.